modules/@ergonode/activity-logs/src/components/Tabs/UsersActivityLogsGridTab.vue

```diff
@@ -19,17 +19,35 @@
 
 <script>
 import ResponsiveCenteredViewTemplate from '@Core/components/Layout/Templates/ResponsiveCenteredViewTemplate';
-import gridFetchDataMixin from '@Core/mixins/grid/gridFetchDataMixin';
+import {
+    DATA_LIMIT,
+} from '@Core/defaults/grid';
+import fetchGridDataMixin from '@Core/mixins/grid/fetchGridDataMixin';
 
 export default {
     name: 'UsersActivityLogsGridTab',
     components: {
         ResponsiveCenteredViewTemplate,
     },
     mixins: [
-        gridFetchDataMixin({
+        fetchGridDataMixin({
             path: 'accounts/log',
         }),
     ],
+    fetch() {
+        return this.onFetchData({
```

Comment:
I think it will be better to create a helper function which will return a default fetch 'settings' object. Something like:

`const getInitialFetchSettings => ({
                offset: 0,
                limit: DATA_LIMIT,
                filters: '',
                sortedColumn: {},
            })`

And then import it in each file and use like:

`this.onFetchData(
getInitialFetchSettings()
)`

I think it will help you in the future when something will change in `onFetchData` function and istead of editing object property in several files, you will change it in one place

Reply:
Thanks, that's good point. I've created default object and added it as default function param